orchestrator/serverless.yml

```diff
@@ -353,4 +353,31 @@ functions:
         - ${file(./config.${self:provider.stage}.json):VPC1}
         - ${file(./config.${self:provider.stage}.json):VPC2}
     events:
-      - schedule: rate(2 minutes)
\ No newline at end of file
+      - schedule: rate(2 minutes)
+
+  get-verfication-fields:
+    warmup: true
+    handler: orchestrator/handlers.verification_handler.get_fields_handler
+    role: ${file(./config.${self:provider.stage}.json):ROLE}
+    tags:
+      Environment: ${file(./config.${self:provider.stage}.json):ENVIRONMENT}
+      Team: ${file(./config.${self:provider.stage}.json):TEAM}
```

Comment:
remove team and owner we haven't configured this in config file

Reply:
https://github.com/singnet/snet-marketplace-service/commit/9e86dd837c177c2c38ca868a948b3f1c93de99e2